packages/gatsby-theme/src/blocks/HeroBlock/index.js

```diff
@@ -33,7 +34,12 @@ const HeroBlock = ({
   }
 
   return (
-    <Hero gutter={gutter} proportion={columnProportion} columns={columns}>
+    <Hero
+      gutter={gutter}
+      proportion={columnProportion}
+      columns={columns}
+      sx={{ textAlign: align }}
```

Comment:
By our conventions, we should not have sx or__css in the blocks implementation. Could we move this to the component?